labs/lab1/chapter1.md

```diff
@@ -18,85 +18,80 @@ Expected completion: 15-20 minutes
 
 Agenda:
 
-* Review Docker and systemd
-* Review Docker help
+* Review docker and systemd
+* Review docker help
 * Explore a Dockerfile
 * Build an image
 * Launch a container
 * Inspect a container
-* Build Docker registry
+* Build docker registry
 
 
-## Docker and systemd
+Perform the following commands as student unless instructed otherwise.
+
+## docker and systemd
 
-Check out the systemd unit file that starts Docker on the CDK and notice that 
-it includes 3 EnvironmentFiles. These files tell Docker how the Docker daemon, 
+Check out the systemd unit file that starts docker on Minishift and notice that 
+it includes 3 EnvironmentFiles. These files tell docker how the docker daemon, 
 storage and networking should be set up and configured. Take a look at those 
 files too. Specifically, in the /etc/sysconfig/docker file check out the registry 
 settings. You may find it interesting that you can `ADD_REGISTRY` and 
 `BLOCK_REGISTRY`. Think about the different use cases for that.
 
-Perform the following commands as student unless instructed otherwise.
-
-XXX we want to consider the new model of using docker from minishift
-in this section. Please run the following command to get access to the
-mihnishift docker
-
 ```bash
-eval $(minishift docker-env)
+minishift ssh "cat /usr/lib/systemd/system/docker.service"
+minishift ssh "cat /usr/lib/systemd/system/docker-storage-setup.service"
+minishift ssh "cat /etc/sysconfig/docker"
+minishift ssh "cat /etc/sysconfig/docker-storage"
+minishift ssh "cat /etc/sysconfig/docker-network"
```

Comment:
should we just `minishift ssh` in first and then run all of the cat commands native?